### ISSUE TITLE
https is broken

### ISSUE THREAD

**Issue by GlenCooper** [mjlamb/BitcoinWhitePaper#3]
http://www.kiwilamb.com/links.html loads ok, but https://www.kiwilamb.com/links.html does not.

I'm a firm believer in that all sites (except .onion URLs) should use https.

What's the deal with your https?  If there's anything I can do to help fix it up, please let me know, even if it means walking you through the process.

**Comment by mjlamb** [comment]
Yes agreed, I need a cert. I'll get that sorted. Domain is also updated to http://www.bitcoinonly.nz/links.html

**Comment by GlenCooper** [comment]
I wouldn't worry about a cert, unless you were doing financial transactions on the site.  For something simple like yours or mine, just use https://letsencrypt.org/

**Comment by mjlamb** [comment]
I use that for my btcpay domain but can you use that with third party host?

**Comment by GlenCooper** [comment]
Yes.  glencooper.com is hosted at hostgator.com and I was able to use letsencrypt to add https to it.  It's been a few years since I did it, so don't remember exactly how, but I know it's possible.
